src/components/partnership/partnership.service.ts

```diff
@@ -494,6 +512,11 @@ export class PartnershipService {
     // mou start and end are now computed fields and do not get updated directly
     const object = await this.readOne(input.id, session);
 
+    this.verifyFundingType(
+      input.fundingType ?? (object.fundingType.value as PartnershipFundingType),
+      input.types ?? (object.types.value as PartnershipType[])
```

Comment:
Looks like I broke the types here when making the secured list changes. Let me see if I can fix.

Reply:
Okay, thx

Reply:
Ok I've got that fix ready in #959. I'll merge it as soon as it goes green. Can you rebase after that and remove these typecasts?

Reply:
Okay, I will.

Reply:
I rebased and I can remove typecast for `types`, but not working with `fundingType`

Reply:
But of course we need to do the same thing for single values lol. #961 up to do that.

Reply:
Ok it's merged!